seonshine_frontend/src/pages/dashboard/index.tsx

```diff
@@ -47,66 +52,129 @@ const Dashboard = () => {
     ],
   };
 
-  const [isModalOpen, setIsModalOpen] = useState<boolean>(false);
-  const [isConfirmModalOpen, setIsConfirmModalOpen] = useState<boolean>(false);
-  const [selectedUser, setSelectedUser] = useState<ChangeStatusPayloadType>();
+  const ITEMS_PER_PAGE = 10;
 
+  const today = format(new Date(), 'yyyy-MM-dd');
   const queryClient = useQueryClient();
+  const {
+    control,
+    watch,
+    formState: { errors },
+  } = useForm<DateSchemaType>({
+    resolver: zodResolver(DateSchema),
+    defaultValues: { date: today },
+  });
 
-  const { currentPage, sortKey, sortType, pageSize, handlePageChange, handleSortingChange, searchField, searchQuery } =
-    useTable({ initPageSize: ITEMS_PER_PAGE, initSortKey: 'user_id' });
+  const watchedDate = watch('date');
 
-  const { data, isFetching } = useGetOrderListApi({
-    [searchField]: searchQuery,
+  const { data: orderList, isFetching } = useGetOrderListDetailApi({
+    params: { date: watchedDate },
   });
 
-  const columns = OrderListHeader();
-  const dishes = [
-    {
-      name: 'Pho',
-      img: 'https://cdn.tgdd.vn/Files/2022/01/25/1412805/cach-nau-pho-bo-nam-dinh-chuan-vi-thom-ngon-nhu-hang-quan-202201250230038502.jpg ',
-    },
-    {
-      name: 'Bun Bo Hue',
-      img: 'https://cdn.tgdd.vn/Files/2018/04/01/1078873/nau-bun-bo-hue-cuc-de-tai-nha-tu-vien-gia-vi-co-san-202109161718049940.jpg',
-    },
-    {
-      name: 'Noodle',
-      img: 'https://cdn.mediamart.vn/images/news/di-mon-vi-hung-dn-lam-mi-kho-xa-xiu-ngon-nhu-ngoai-hang_16bbe0f5.jpg',
-    },
-    {
-      name: 'Banh mi Sai Gon',
-      img: 'https://static.vinwonders.com/production/banh-mi-sai-gon-2.jpg',
-    },
-    {
-      name: 'Grill Pork Rice',
-      img: 'https://i.ytimg.com/vi/h__kLq8NG2I/hq720.jpg?sqp=-oaymwEhCK4FEIIDSFryq4qpAxMIARUAAAAAGAElAADIQj0AgKJD&rs=AOn4CLDqn7vasJHB1JVJB8uobiB67rxztw',
-    },
+  const columns = OrderListHeader;
+  const data: OrderListType[] = orderList
+    ? orderList.data.map((order) => ({
+        restaurant_name: order.restaurant_name,
+        employee_name: order.username,
+        ordered_items: order.item_name,
+        date: order.submitted_time,
+      }))
+    : [];
+
+  const { data: todayMenuList } = useGetTodayMenuListApi({ enabled: true });
+  const { data: dashboardSummary } = useGetDashBoardSummary({ enabled: true });
+  const { data: orderPeriod } = useGetOrderPeriodApi();
+
+  const navigate = useNavigate();
+  const handleNavigate = () => {
     {
-      name: 'Bun Dau Mam Tom',
-      img: 'https://vietnamnomad.com/wp-content/uploads/2023/05/What-is-bun-dau-mam-tom.jpg',
-    },
-  ];
+      currentUser?.role_id === RoleEnum.USER ? navigate('/order-menu', { state: { tab: 1 } }) : navigate('/order');
```

Comment:
define order tab enum with meaning name instead of using tab index. Ex: `OrderTabEnum {OrderMenu, OrderHistory}`